src/SonarScanner.MSBuild.PreProcessor/Roslyn/ActiveRule.cs

```diff
@@ -40,20 +42,20 @@ public SonarRule()
         {
         }
 
-        public SonarRule(string repoKey, string ruleKey)
+        public SonarRule(string ruleKey, string repoKey)
         {
             RepoKey = repoKey;
             RuleKey = ruleKey;
         }
 
-        public SonarRule(string repoKey, string ruleKey, bool isActive)
+        public SonarRule(string ruleKey, string repoKey, bool isActive)
         {
             RepoKey = repoKey;
             RuleKey = ruleKey;
             IsActive = isActive;
         }
 
-        public SonarRule(string repoKey, string ruleKey, string internalKey, bool isActive)
+        public SonarRule(string ruleKey, string repoKey, string internalKey, bool isActive)
```

Comment:
Switched the arguments to be compatible with the `SonarQubeRule` class that comes from the client (hence many of the changes)